packages/codefixes/src/fixes/addMissingTypesBasedOnInheritance/index.ts

```diff
@@ -39,27 +39,24 @@ export class AddMissingTypesBasedOnInheritanceCodeFix implements CodeFix {
           continue;
         }
 
-        const ancestorMethodType = checker.getTypeOfSymbolAtLocation(
-          ancestorMethodSymbol,
-          ancestorMethodSymbol.valueDeclaration
-        );
-
-        const ancestorMethodCallSignatures = ancestorMethodType.getCallSignatures();
+        const ancestorMethodCallSignatures = checker
+          .getTypeOfSymbolAtLocation(ancestorMethodSymbol, ancestorMethodSymbol.valueDeclaration)
+          .getCallSignatures();
 
         // Don't support multiple signatures
-        if (ancestorMethodCallSignatures.length > 1) {
+        if (ancestorMethodCallSignatures.length !== 1) {
           continue;
         }
 
         const parameterSymbol = ancestorMethodCallSignatures[0].parameters[currentParameterIndex];
 
         // Additional parameter in a child function won't have param in parent function signature
-        if (!parameterSymbol) {
+        if (!parameterSymbol || !parameterSymbol.valueDeclaration) {
```

Comment:
Covers if there is no `valueDeclaration` available for some magical reason

Reply:
Any chance you have a test case we can add for this?

Reply:
https://github.com/microsoft/TypeScript/issues/36697 :)